tests/test_queries.py

```diff
@@ -819,6 +837,80 @@ def validate(self, expected_values=None):
         return True
 
 
+class QueryTestDatasetMappingPlan(QueryTestBase):
+    NAME = "dataset_mapping_plan"
+
+    def __init__(self, handle_data_skew, persist, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._handle_data_skew = handle_data_skew
+        self._persist = persist
+
+    def make_query(self) -> None:
+        self._model = ProjectQueryModel(
+            name=self.NAME,
+            project=ProjectQueryParamsModel(
+                project_id=self.get_project_id(),
+                include_dsgrid_dataset_components=False,
+                dataset=DatasetModel(
+                    dataset_id="tempo_mapped",
+                    source_datasets=[
+                        StandaloneDatasetModel(dataset_id="tempo_conus_2022"),
+                    ],
+                ),
+                mapping_plan=[
+                    DatasetMappingPlan(
```

Comment:
Question in a strange place because I missed it above: When does time mapping happen?